source/core_http_client.c

```diff
@@ -924,12 +940,8 @@ static int httpParserOnBodyCallback( llhttp_t * pHttpParser,
 
     /* The next location to write. */
 
-    /* MISRA Rule 11.8 flags casting away the const qualifier in the pointer
-     * type. This rule is suppressed because when the body is of transfer
-     * encoding chunked, the body must be copied over the chunk headers that
-     * precede it. This is done to have a contiguous response body. This does
-     * affect future parsing as the changed segment will always be before the
-     * next place to parse. */
+    /* MISRA Ref 11.8.1 [Function pointer and use of const pointer] */
```

Comment:
There's no function pointer involved here.

Reply:
The p$Response->[pBody](https://github.com/FreeRTOS/coreHTTP/blob/main/source/include/core_http_client.h#L486) is a ( const uint8_t *)  so the cast to just a char * removes the const qualifier, setting off the 11.8 rule.